cli/src/lib.rs

```diff
@@ -1950,10 +1953,55 @@ fn validator_flags(cfg: &WithPath<Config>) -> Result<Vec<String>> {
                         flags.push(entry["filename"].as_str().unwrap().to_string());
                     }
                 } else if key == "clone" {
+                    // Client for fetching accounts data
+                    let client = if let Some(url) = entries["url"].as_str() {
+                        RpcClient::new(url.to_string())
+                    } else {
+                        return Err(anyhow!(
+                    "Validator url for Solana's JSON RPC should be provided in order to clone accounts"
```

Comment:
some clearer wording here would be good clarifying that this should be the url where the accounts are _cloned from_ and not the url where the tested program lives

answering the question  _which_ url, the url of the cluster where the test happens or the url where the accounts are cloned from